src/css/styles/_layout.scss

```diff
@@ -1,9 +1,3 @@
-.l-container-padding {
-  padding-right: 4rem;
-  padding-left: 4rem;
-
-  @include media-breakpoint-down(md) {
-    padding-right: 2rem;
-    padding-left: 2rem;
-  }
+.l-no-padding {
+  padding: 0;
```

Comment:
- [ ] `layout` or `helper`?

Reply:
It's a layout helper. For me can be both `h-*` and `l-*`,